docs/sdk/database/aliases.md

```diff
@@ -0,0 +1,114 @@
+---
+title: Table aliases
+description: Simplify development using table aliases instead of raw auto-generated table names.
+---
+
+Tableland tables are created in a globally unique format. However, this can disrupt the development workflow as it requires developers to adhere to the exact table name, which can be cumbersome to remember and type out. Instead, you can leverage table aliases to enhance the development experience and configure custom table names used locally by the SDK.
+
+## Overview
+
+When instantiating a `Database`, the `aliases` parameter can be used to configure table aliases within a JSON file. The `aliases` parameter should make use of the SDK's `helpers.jsonFileAliases('...')` method where the configuration file's path is passed to it. Table reads and writes will automatically read from this file upon `Database.prepare(...)` method calls.
+
+An alias is, in a sense, the same as a table prefix but with local uniqueness restrictions and only scoped to your project alone (i.e., has no relationship to the broader Tableland network).
+
+Under the hood, `aliases` contains an object with `read` and `write` functions. The `read` function must return a mapping of the project-scoped table alias to the universally unique Tableland table nam—for example, `{ "users": "users_31337_2" }`. The `write` function will be called any time a `Database` instance is used to create a table and modifies the JSON file, writing the mapping to it.
+
+:::note
+All project aliases must be unique. If you try to create a table with a preexisting alias saved in your config file, you'll run into an error: `table name already exists in aliases`.
+:::
+
+## Configuration file
+
+As a best practice, the alias config file should exist in your project's directory to ensure the mappings from table aliases to the generated names are persisted and do not get lost. First, create a JSON file with an empty object (`{}`) in the root of your project (or a subdirectory). It will ultimately contain all of your table aliases. Let's name the file `tableland.config.json`, but it can be any file name you'd like.
+
+```sh
+echo '{}' > tableland.config.json
+```
+
+Then, import `jsonFileAliases` from the SDK's `helpers` into your source code; this is a utility function that allows you to simply provide the path to your aliases file.
+
+```js
+import { Database } from "@tableland/sdk";
+import { jsonFileAliases } from "@tableland/sdk/helpers";
+
+const db = new Database({
+  signer,
+  // highlight-start
+  // Specify the path to your aliases config file
+  aliases: jsonFileAliases("./tableland.config.json"),
```

Comment:
Should we name the example file something else, so there's not a potential conflict with the config files used by the CLI?
maybe `tableland.aliases.json`?